### PR TITLE
feat: see stickers details when clicking on a sticker

### PR DESCRIPTION
**Pull request by richard-ramos**
Displays a popup containing all the packs that belong to a sticker pack
This option is only available after all the stickers packs have been loaded. 

 
image

Video:
https://user-images.githubusercontent.com/1106587/110371347-3b2dcc80-8023-11eb-9d50-209dfcf25ee0.mp4

### CODE REVIEW

File: ui/shared/status/StatusStickerPackClickPopup.qml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    property string author: ""
+    property string price: ""
+    property bool installed: false;
+    property bool bought: false;
+    property bool pending: false;
+    property var stickers;
+
+    Component.onCompleted: {
+        const idx = chatsModel.stickers.stickerPacks.findIndexById(packId, false);
+        if(idx == -1) close();
```
Comment (jrainville) [inline ui/shared/status/StatusStickerPackClickPopup.qml:26]:
```suggestion
        if(idx === -1) close();
```

**Comment by richard-ramos** [comment]
> Looks good, though maybe I'm missing something, but you don't seem to open `statusStickerPackClickPopup` anywhere. Did you forget to push a file? lol

I moved it to ChatLayout with the idea of reusing it inside the standard sticker popup, but forgot to do the change there.

**Comment by iurimatias** [comment]
@richard-ramos it doesn't work for me

**Comment by iurimatias** [comment]
issue #1062

**Comment by iurimatias** [comment]
https://user-images.githubusercontent.com/176720/111369778-d2bb9c80-866d-11eb-98ac-70010087732a.mov

**Pull request merged**